From 204cfffefb2ee6058e5d22a4b65cff22e6cff5ee Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Thu, 16 Jul 2015 21:30:02 +0200
Subject: [PATCH 2/2] Include linux/sockios.h to fix musl build

Including <sys/ioctl.h> and <net/if.h> is not sufficient
to musl to find SIOCSIFNAME, so <linux/sockios.h> must be
included too.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 slcan_attach.c | 1 +
 slcand.c       | 1 +
 2 files changed, 2 insertions(+)

diff --git a/slcan_attach.c b/slcan_attach.c
index ab8ff0a..485cbe3 100644
--- a/slcan_attach.c
+++ b/slcan_attach.c
@@ -52,6 +52,7 @@
 #include <net/if.h>
 #include <termios.h>
 #include <linux/tty.h>
+#include <linux/sockios.h>
 
 void print_usage(char *prg)
 {
diff --git a/slcand.c b/slcand.c
index e5e4752..e8c70e0 100644
--- a/slcand.c
+++ b/slcand.c
@@ -38,6 +38,7 @@
 #include <net/if.h>
 #include <termios.h>
 #include <linux/tty.h>
+#include <linux/sockios.h>
 
 /* Change this to whatever your daemon is called */
 #define DAEMON_NAME "slcand"
-- 
2.1.4

